src/suites/cts/validation/vertex_input.spec.ts

```diff
@@ -575,37 +669,47 @@ g.test('check attribute offset out of bounds', async t => {
   }
 });
 
-g.test('check attribute offset overflow', async t => {
+g.test('check multiple of 4 bytes constraint on offset', async t => {
   const vertexInput: GPUVertexInputDescriptor = {
     vertexBuffers: [
       {
         stride: 0,
         attributeSet: [
           {
-            offset: Number.MAX_SAFE_INTEGER,
+            offset: Float32Array.BYTES_PER_ELEMENT,
             shaderLocation: 0,
             format: 'float',
           },
         ],
       },
     ],
   };
-  const descriptor = t.getDescriptor(vertexInput, VERTEX_SHADER_CODE_WITH_NO_INPUT);
-
-  await t.expectValidationError(() => {
+  {
+    // Control case, setting offset 4 bytes
+    const descriptor = t.getDescriptor(vertexInput, VERTEX_SHADER_CODE_WITH_NO_INPUT);
     t.device.createRenderPipeline(descriptor);
-  });
+  }
+  {
+    // Test offset not multiple of 4 bytes
+    vertexInput.vertexBuffers![0].attributeSet![0].offset = 2;
```

Comment:
The size of a float is also 4, so it's conceivable that the offset would have to be a multiple of 4 for float, and a multiple of 2 for uchar2. Need to test with format=uchar2 and offset=2. Should that be invalid?

(If yes, that would seem unfortunate because it forces dead padding between values.)

Reply:
I've set format to `uchar2` and it is actually valid.
This means I have to update https://dawn-review.googlesource.com/c/dawn/+/11902 to not consider format as well. 
I've updated test.

Reply:
Argh! Even though https://developer.apple.com/documentation/metal/mtlvertexattributedescriptor/1515785-offset says "The offset value must be a multiple of 4 bytes.", it is not raising an error with offset set to 2 and format set to "uchar2".

@Kangz @austinEng What shall we do?

Reply:
The documentation says so, so we should enforce it for now (or just leave it as is until we figure out the answer). Hopefully we can figure out the answer on the gpuweb thread.

Reply:
Found a good source from a glTF issue:
https://github.com/KhronosGroup/glTF-Validator/issues/10#issuecomment-269799846

Let's enforce the 4 byte alignments.

Reply:
Done.